src/components/TheHeader.vue

```diff
@@ -136,36 +239,78 @@ export default {
     .button {
       margin-left: 1em;
     }
+  }
 
-    .menu__toggle {
-      height: 100%;
-      width: 60px;
-      cursor: pointer;
-      display: none;
-      justify-content: center;
-      align-self: center;
+  .menu__toggle {
+    height: 100%;
+    width: 60px;
+    cursor: pointer;
+    display: none;
+    justify-content: center;
+    align-items: center;
 
+    &__lines {
       .line {
         width: 32px;
         height: 3px;
         background-color: $tuna;
         margin-bottom: 5px;
       }
+
+      .line1 {
+        transform-origin: 0% 0%;
+        transition: transform 0.4s ease-in-out;
+      }
+
+      .line2 {
+        transition: transform 0.2s ease-in-out;
+      }
+
+      .line3 {
+        transform-origin: 0% 100%;
+        transition: transform 0.4s ease-in-out;
+      }
+    }
+    &__lines--active {
+      .line {
+        width: 32px;
+        height: 3px;
+        background-color: $tuna;
+        margin-bottom: 5px;
+      }
```

Comment:
I think this is duplicated from above, only the styles belo are needed 😄

Reply:
hmmmm I have added this class knowing it is duplicated - without it when button is --active .line class does not get applied to the element and therefore lines are not visible 😿 ?

Reply:
The class on it's own is fine! It's just that we have duplicated the `.line { }` styles on their own (look a few lines above) - the styles below which modify the behavior when "active" are fine though 😄